Dockerfile

```
@@ -0,0 +1,19 @@
+FROM python:3.12-alpine AS base
+ARG ANSIBLE_VERSION=10.0
+RUN apk -U upgrade --available &&\
+    # Required to install ansible pip package, bear in mind to remove those build deps at the end of this RUN directive
+    apk add --no-cache gcc musl-dev libffi-dev openssl-dev &&\
+    # Add here package mandatory to be able to run ansible
+    apk add --no-cache openssh-client ca-certificates&&\
+    pip install --no-cache-dir --upgrade pip &&\
+    pip install --no-cache-dir ansible==${ANSIBLE_VERSION}.* ansible-runner~=2.4 &&\
+    # Cleanup package manager cache and remove build deps
+    rm -rf /var/cache/apk/* &&\
+    pip cache purge &&\
+    apk del gcc musl-dev gcc musl-dev libffi-dev openssl-dev
+
+FROM base AS gcp
+RUN pip install --no-cache-dir requests==2.32.3 google-auth==2.32.0
```

these require constant maintenance, I'm not a huge fan. if we insist, I'd do at least:
```suggestion
RUN pip install --no-cache-dir requests==2.* google-auth==2.*
```
same for `boto3`